pybricksdev/cli/__init__.py

```diff
@@ -262,6 +262,7 @@ async def reconnect_hub():
                 return hub
 
             response_options = [
+                "Change Target File",
```

Comment:
Don't we want to have the most common option first? It seems like this one should be 3rd.